src/components/HierarchyTable/HierarchyTable.tsx

```diff
@@ -232,6 +232,7 @@ export const HierarchyTable: FunctionComponent<HierarchyTableProps> = (props) =>
                         widgetName={props.meta.name}
                         widgetFieldMeta={item}
                         readonly
+                        customProps={{hierarchyDepth: indentLevel + 1}}
```

Comment:
This should memoized or `<Field />` will be recreated every render

Reply:
Fixed.